pyproject.toml

```diff
@@ -0,0 +1,40 @@
+[tool.poetry]
+name = "project-liberation"
+version = "v1.10.0"
+description = "Necessary packages for Project Liberation"
+authors = ["Karol Beker <karol.beker@codepoets.it>"]
+
+[tool.poetry.dependencies]
+python = "^3.7"
```

Comment:
Let's change it to `python = "3.7.*"` like we had to on my machine, so it's always use newest 3.7 version.

Reply:
Done